src/style.css

```diff
@@ -1,3 +1,88 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(./font/Roboto-Medium.ttf);
+}
+
+body {
+  font-family: Roboto, Arial, sans-serif;
+  background-color: gray;
+  display: flex;
+  justify-items: center;
+  font-size: 12px;
+}
+
+header {
+  width: 100%;
+  background-color: rgba(255, 255, 255, 1);
+  text-align: center;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo img {
+  height: 40px;
+  width: 40px;
+}
+
+nav ul {
+  list-style: none;
+  text-align: center;
+  display: flex;
+  height: 60px;
+}
+
+nav ul li {
+  position: relative;
+}
+
+nav ul li a {
+  color: black;
+  text-decoration: none;
+  margin: 0 10px;
```

Comment:
A link should not contain a margin. Try adding margin-right to each li element, instead of a first one. It will probably solve all problems with tests :)

Reply:
Próbowałem dodać margines nakładając go na klasę nav_list ale nie spełniało to założeń więc musiałem wrócić do pierwotnej wersji wysłanego wcześniej kodu.